pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -592,6 +592,13 @@ func (o *OsqueryInstance) StartOsqueryExtensionManagerServer(name string, socket
 				"err", err,
 				"extension_name", name,
 			)
+			// PR Note, remove after test working
+			// On windows, during test this always returns error
+			// "status 1 registering extension: Failed adding registry: SQLITE_CANTOPEN"
+			// the source of the error is in osquery-go code
+			// https://github.com/osquery/osquery-go/blob/master/server.go#L217
+			// coming out of
+			// https://github.com/osquery/osquery-go/blob/master/gen/osquery/osquery.go#L2225
 			return fmt.Errorf("running extension server: %w", err)
```

Comment:
always hits error

```
"status 1 registering extension: Failed adding registry: SQLITE_CANTOPEN"
```

Reply:
This is coming out of osquery. https://github.com/osquery/osquery/blob/dcd8594f8d8687cb13bc9540b5aad2c26c5f8d9e/osquery/extensions/interface.cpp#L147

Extensions register themselves to osquery, which sticks them into the registry. `SQLITE_CANTOPEN` smells a lot like some systemy permissiony issue. Do these tests work on a windows machine? (eg: is it a function of CI)

Reply:
it does not work on my windows machine, but it does work on CI .... I will investigate the permissions on my windows machine

Reply:
it works when you run as admin